makefile

```diff
@@ -4,7 +4,7 @@ TEST_DIRS     ?= $(shell sh -c "find $(TOP_SRC_DIRS) -name \\*_test.go \
                    -exec dirname {} \\; | sort | uniq")
 BIN_DIR := $(GOPATH)/bin
 SHELL = /bin/bash
-BINARY = metrics
+BINARY ?= metrics
```

Comment:
Why the `?=` ?

Reply:
The `?=` allows it to be overriden by an env var, otherwise its a fixed value.